config/routes.rb

```diff
@@ -15,6 +15,7 @@
   namespace :admin do
     resources :users
     resources :people
+    resources :comments
```

Comment:
```
resources :comments
```
creates seven different routes but you only need the index route!

Please change this to 

```
resources :comments, only: [:index]
```

See http://guides.rubyonrails.org/routing.html for more details!
@raluka